src/features/place-bid/PlaceBidButton/PlaceBidButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+
+import { usePlaceBidData } from '../usePlaceBidData';
+
+import { PlaceBidModal } from '..';
+
+type PlaceBidButtonProps = {
+	zna: string;
+	isRoot?: boolean;
```

Comment:
nit: would probably make for a clearer component if you changed `isRoot` to `trigger`. At the moment, if you're rendering the `PlaceBidButton`, it's not obvious what `isRoot` does. Not a big issue though and definitely not necessary for this PR!

Reply:
updated in follow up pr